python_modules/libraries/dagster-polars/dagster_polars/io_managers/type_routers.py

```diff
@@ -14,10 +14,10 @@
     get_origin,
 )
 
-if sys.version_info >= (3, 9):
```

Comment:
Mind explaining this one a bit more? Is the idea that 3.9.1 doesn't have the thing we need here?

Reply:
exactly yep -- that only got added in py 3.10